src/app/admin/core/authorship.service.ts

```diff
@@ -16,11 +16,9 @@ export class AuthorshipService {
 
   async changeAuthorship(oldAuthor: User, id: string, newAuthor: string) {
     return this.http
-    .post(ADMIN_ROUTES.CHANGE_AUTHOR(oldAuthor.username, id),
+    .post(ADMIN_ROUTES.CHANGE_AUTHOR(oldAuthor.id, id),
     {
-      'fromUserID': oldAuthor.id,
-      'toUserID': newAuthor,
-      'objectID': id,
+      'author': newAuthor,
```

Comment:
So this post updates the author  to newAuthor, for the removed author, shouldn't there be some update that removes this object from that user being the author?

Reply:
@mfranz1 So that is done completely on the backend. When we switch an author we are switching who has the ability to edit it in an unreleased state.